core/modules/system/system.install

```diff
@@ -98,6 +100,21 @@ function system_requirements($phase) {
     $requirements['php_register_globals']['value'] = $t('Disabled');
   }
 
+  // Test PHP max_input_vars setting.
+  $max_input_vars = ini_get('max_input_vars');
+  $requirements['php_max_input_vars'] = array(
+    'title' => $t('PHP form input max'),
+    'value' => $t("PHP limits the number of form values to @value items.", array('@value' => number_format($max_input_vars, 0, '.', ','))),
+  );
+  // 1000 is the default value, and indicates potential issues.
+  if ($max_input_vars == 1000) {
+    $requirements['php_max_input_vars']['description'] = $t('When set too low, PHP\'s <code>max_input_vars</code> setting can cause silent failures on large forms. Use caution when using the permissions page, ordering large menus, arraging large vocabularies, or editing content with lots of fields.');
```

Comment:
```suggestion
    $requirements['php_max_input_vars']['description'] = $t('When set too low, PHP\'s <code>max_input_vars</code> setting can cause silent failures on large forms. Use caution when using the <em>Permissions</em> page, ordering large menus, arranging large vocabularies, or editing content with lots of fields.');
```

Reply:
I don't see a reason for emphasizing permissions (?) but I've made the other change.

Reply:
The recommendation is as per other places in the UI, where we render things that have a name/title in italics. It's not "some permissions page"; it's a "the _Permissions_ page".